File: 프로그래머스/0/120850. 문자열 정렬하기 （1）/문자열 정렬하기 （1）.py
def solution(my_string):
    answer = []
    
    for i in range(len(my_string)) :
        for j in range(0, 10) :
            if my_string[i : i+1] == str(j) :
                answer.append(int(my_string[i : i+1]))
    answer.sort()        
    return answer

File: 프로그래머스/0/120850. 문자열 정렬하기 （1）/문자열 정렬하기 （1）.js
function solution(my_string) {
    var answer = [];
    // answer = my_string.split('')
    //                   .filter(char => !Number.isNaN(Number(char)))
    //                   .map(Number)
    //                   .sort((a,b) => a - b);
    for (let char of my_string) {
        const num = Number(char);
        if (!Number.isNaN(num)) {
            answer.push(num);
        }
    }
    answer.sort((a, b) => a - b);
    return answer;
}
